variants/trackerd/platformio-custom.py

```diff
@@ -0,0 +1,77 @@
+# trunk-ignore-all(ruff/F821)
+# trunk-ignore-all(flake8/F821): For SConstruct imports
+import sys
```

Comment:
I'm not sure why a custom platformio script would be needed for this board? It's a standard esp32 target

Reply:
with bin/platformio-custom.py the dragino goes in bootloop.

_Guru Meditation Error: Core  0 panic'ed (Cache disabled but cached memory region accessed).
Backtrace: 0x402097d5:0x3ffe3b10 0x4008284b:0x3ffe3b50 0x4013199d:0x3ffe3b80 0x40082941:0x3ffe3ba0 0x401319ff:0x3ffe3bd0 0x401361f0:0x3ffe3c20 0x400839dd:0x3ffe3c50 0x40079306:0x3ffe3c90
Guru Meditation Error: Core  0 panic'ed (Cache disabled but cached memory region accessed)._

my version fix the problem
I spend 2 days to understand why, i suspect there is something wrong with the settings provided by the python script.
In order to not touch original script, i rewrite it in variants.
Dragino has esp32 pico D4, maybe slighty different from other esp32 ?

![pcb-photo](https://github.com/meshtastic/firmware/assets/8080238/7e2b1fa9-6fde-425c-a556-ecc950763f19)

Reply:
check without extra_scripts = variants/trackerd/platformio-custom.py - work well 
before also have  bootloop

Reply:
@fzellini  can you check your build with latest code without extra_scripts ?